src/KOKKOS/kokkos.h

```diff
@@ -32,6 +32,7 @@ class KokkosLMP : protected Pointers {
   int num_threads,ngpu;
   int numa;
   int auto_sync;
+  int gb_test;
```

Comment:
@stanmoore1 , Can this flag be eliminated, and just go with a sane default?  I think the new code that this controls is likely to be better overall, but I didn't have a chance to look closely.  Thanks!

Reply:
I got rid of this flag and just use an `#ifdef` on CUDA